Library/ViewModels/PledgePaymentMethodsViewModel.swift

```diff
@@ -110,19 +100,46 @@ public final class PledgePaymentMethodsViewModel: PledgePaymentMethodsViewModelT
     )
     .map { ($0.0, $0.1, $0.2, false) }
 
-    let newSetupIntentCards = self.newStripeIntentCreditCardProperty.signal.skipNil()
-      .map { data -> [PaymentSheetPaymentMethodCellData] in
-        let (displayData, setupIntent) = data
+    let newlyAddedCardProducer = self.newStripeIntentCreditCardProperty.signal.skipNil()
```

Comment:
This is the beefiest part of the change - when a new card is selected, add it to the Kickstarter backend, then populate the payment sheet with the actual `UserCreditCards.CreditCard` returned by that mutation.